trade_remedies_public/templates/dashboard.html

```diff
@@ -92,30 +92,44 @@ <h1 class="heading-medium">Your draft registrations of interest</h1>
                     </div>
                     <!-- 3rd party invites -->
                     <div class="margin-top-2">
-                        {% if invite_submissions %}
-                            <h1 class="heading-medium">Your 3rd Party invites</h1>
-                            <UL>
-                                {% for invite in invite_submissions %}
-                                    {% if not invite.invitations.0.deleted_at %}
-                                        <LI>
-                                            {% if 'FEATURE_FLAG_INVITE_JOURNEY' in user.groups and invite.status.draft == True %}
-                                                <a href="{% url 'invite_representative_task_list_exists' invitation_id=invite.invitations.0.id %}">{{ invite.case.reference }}
-                                                    - {% format_date invite.created_at '%d %b %Y' %}</a>
-                                            {% else %}
-                                                <a href="/case/{{ invite.case.id }}/submission/{{ invite.id }}/">
-                                                    {{ invite.case.reference }}
-                                                    - {% format_date invite.created_at '%d %b %Y' %}
-                                                </a>
-                                            {% endif %}
-                                            ({{ invite.name }} - {{ invite.status.name }})
-                                        </LI>
-                                    {% endif %}
-                                {% endfor %}
-                            </UL>
+                        {% if is_org_owner or pending_invitation_count or pending_invitation_deficient_docs_count or pending_invitation_waiting_trs_approval_count %}
```

Comment:
org_owner needs to be always True, so 
```suggestion
                        {% if is_org_owner and pending_invitation_count or pending_invitation_deficient_docs_count or pending_invitation_waiting_trs_approval_count %}
```